client/src/components/Icon/Icon.tsx

```diff
@@ -13,13 +13,26 @@ const Icon: React.FunctionComponent<IconProps> = ({
   name,
   className,
   title,
-}) => (
-  <>
-    <svg className={`icon icon-${name} ${className || ''}`} aria-hidden="true">
-      <use href={`#icon-${name}`} />
-    </svg>
-    {title && <span className="visuallyhidden">{title}</span>}
-  </>
-);
+}) => {
+  if (name && name.startsWith('M')) {
+    return (
+      <svg className={`icon ${className || ''}`} aria-hidden="true">
+        <path d={name} />
```

Comment:
```suggestion
        <path d={name.join(' ')} />
```

Assuming we have an Array, we need to convert it back to a string. I'm not sure if we should add the space to join but maybe the documentation could be updated with another example.

Reply:
I will test this out and make the correct documentation update.

Reply:
@lb- I have tried this, it doesn't work. Literally, we are like back to square one with the same problem I am trying to resolve. Like the solution I currently have, it only work if `icon` is directly set to string. 
![fHklT4048v](https://github.com/wagtail/wagtail/assets/87539866/d1b34446-04e8-48fa-887d-26ef4a5d528e)

Reply:
It looks you you're rendering the string as is. Are you sure you are passing in the path as an array via the config?

Reply:
I have created a seperate props for `svgPath`, let me know what you think of the corrections I made. This works, and I think it might be easier managing the `svgPath` data by using its own seperate props rather than getting from `name` props.